File: BackEnd-Siukat/models/log_wali.go
package models

import "time"

type LogWali struct {
	IDLogWali       int        `gorm:"primaryKey;column:id_log_wali" json:"id_log_wali"`
	NoPeserta       string     `gorm:"column:no_peserta;type:varchar(255)" json:"no_peserta"`
	StatusWali      string     `gorm:"column:status_wali;type:enum('ada','tidak')" json:"status_wali"`
	NamaWali        string     `gorm:"column:nama_wali;type:varchar(255)" json:"nama_wali"`
	AlamatWali      string     `gorm:"column:alamat_wali;type:text" json:"alamat_wali"`
	ProvinsiWali    string     `gorm:"column:provinsi_wali;type:varchar(255)" json:"provinsi_wali"`
	KabkotWali      string     `gorm:"column:kabkot_wali;type:varchar(255)" json:"kabkot_wali"`
	KecamatanWali   string     `gorm:"column:kecamatan_wali;type:varchar(255)" json:"kecamatan_wali"`
	PekerjaanWali   string     `gorm:"column:pekerjaan_wali" json:"pekerjaan_wali"`
	KesanggupanWali int        `gorm:"column:kesanggupan_wali" json:"kesanggupan_wali"`
	ScanWali        string     `gorm:"column:scan_wali;type:varchar(255)" json:"scan_wali"`
	Atribut         string     `gorm:"column:atribut;type:enum('original','sanggah')" json:"atribut"`
	Executor        string     `gorm:"column:executor;type:varchar(255)" json:"executor"`
	Timestamp       *time.Time `gorm:"column:timestamp;type:datetime" json:"timestamp"`
}

func (LogWali) TableName() string {
	return "log_wali"
}

File: ZhammDataBeLama/siukat-service-dev_snmptn_2022/models/log_wali.js
var Sequelize = require('sequelize')
var db = require('../config/database')

const LogWali = db.define('log_wali', {
  id_log_wali:{
    type: Sequelize.INTEGER,
    primaryKey: true
  },
  no_peserta: {
    type: Sequelize.STRING
  },
  status_wali: {
    type: Sequelize.ENUM('ada', 'tidak')
  },
  nama_wali: {
    type: Sequelize.STRING
  },
  alamat_wali: {
    type: Sequelize.STRING
  },
  provinsi_wali: {
    type: Sequelize.INTEGER
  },
  kabkot_wali: {
    type: Sequelize.INTEGER
  },
  kecamatan_wali: {
    type: Sequelize.INTEGER
  },
  kesanggupan_wali: {
    type: Sequelize.INTEGER
  },
  scan_wali: {
    type: Sequelize.STRING
  },
  atribut: {
    type: Sequelize.ENUM('original', 'sanggah')
  },
  executor: {
    type: Sequelize.STRING
  },
  timestamp: {
    type: Sequelize.DATE
  }
},
 {
  // don't add the timestamp attributes (updatedAt, createdAt)
  timestamps: false,

  // don't delete database entries but set the newly added attribute deletedAt
  // to the current date (when deletion was done). paranoid will only work if
  // timestamps are enabled
  paranoid: true,

  // don't use camelcase for automatically added attributes but underscore style
  // so updatedAt will be updated_at
  underscored: true,

  // disable the modification of tablenames; By default, sequelize will automatically
  // transform all passed model names (first parameter of define) into plural.
  // if you don't want that, set the following
  freezeTableName: true,

  // define the table's name
  tableName: 'log_wali'
}
);
module.exports = LogWali
